ebos/eclgenericwriter.cc

```diff
@@ -28,6 +28,7 @@
 #include <opm/grid/cpgrid/GridHelpers.hpp>
 #include <opm/grid/polyhedralgrid.hh>
 #include <opm/grid/utility/cartesianToCompressed.hpp>
+#include "eclalugridvanguard.hh"
```

Comment:
There are still compile errors:
```

In file included from ../../ebos/eclalugridvanguard.hh:32,
                 from ../../ebos/eclgenericwriter.cc:31:
../../ebos/alucartesianindexmapper.hh:33:10: fatal error: dune/alugrid/grid.hh: No such file or directory
   33 | #include <dune/alugrid/grid.hh>
      |          ^~~~~~~~~~~~~~~~~~~~~~
```
Please move this include inside `#if HAVE_DUNE_ALUGRID`, to resolve them.